app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -57,6 +61,9 @@ class ScanSimpleIdScreen(context: Context, attrs: AttributeSet) : ConstraintLayo
     toolBar.setNavigationOnClickListener { screenRouter.pop() }
     setupShortCodeTextField()
 
+    qrCodeScannerView.visibleOrGone(config.useNewQrScanner)
+    qrCodeScannerViewOld.visibleOrGone(!config.useNewQrScanner)
```

Comment:
Since the view is present in the layout, won't this activate the camera in both the views?

I was thinking we should manually inflate one or the other view based on the flag.

Reply:
I don't think it will activate the camera views If the view is set to gone it won't inflate the views right?

Reply:
The views will always be inflated regardless of the visibility status.